packages/cardano-services/src/TxSubmit/TxSubmitHttpService.ts

```diff
@@ -46,7 +46,9 @@ export class TxSubmitHttpService extends HttpService {
           } catch {
             isHealthy = false;
           }
-
+          if (error instanceof ProviderError) {
```

Comment:
Where does the error type come from? when I add this to the CardanoNodeErrors, I eventually run into an issue they come from another library and they aren't added to a mapper but it doesn't seem like the mapper would accept it either.

Reply:
We’re currently only expecting _submission_ errors, the ones returned by Cardano Node, to be thrown by the provider, but we should change that to expect a new type of error `PreSubmissionErrors` in addition. These errors can be defined in the `core` package`

Reply:
0bc4fd1a9ab89f5e229feace62c4b0e3d1485b9c adds the presubmission errors 
 ebc0237c005b9b050f038dcac487ca1e9a5875e6 addresses this line.